packages.gradle

```diff
@@ -102,7 +102,7 @@ def isValidMavenBuildThreads(threads) {
 }
 
 def getPkgNameSuffix(component, type) {
-    def originalSuffix = type.equalsIgnoreCase("deb") ? component.deb_pkg_suffix : component.rpm_pkg_suffix
+    def originalSuffix = type.equalsIgnoreCase("deb") ? component.deb_pkg_suffix?:"" : component.rpm_pkg_suffix?:"%{nil}"
```

Comment:
@Kapkiai I feel chaining ternary operator and elvis operator spoils readability. How about splitting the line and adding whitespace around the `?:` operator? like:

```
type.equalsIgnoreCase("deb") ?
    component.deb_pkg_suffix ?: "" :
    component.rpm_pkg_suffix ?: "%{nil}"
```

Reply:
@iwasakims good suggestion. I have made changes.

Reply:
@iwasakims is it a good path to refactor the changes as below?

``` groovy
def defaultSuffix = type.equalsIgnoreCase("deb") ?  "" : "%{nil}"
def originalSuffix = type.equalsIgnoreCase("deb") ?
            component.deb_pkg_suffix ?: defaultSuffix :
            component.rpm_pkg_suffix ?: defaultSuffix
```

With that, we will have the defaults defined once.

Reply:
I think we should avoid extra variable definition if it is used only here.